src/qml/FeatureForm.qml

```diff
@@ -599,6 +600,7 @@ Page {
               property bool isAdding: form.state === 'Add'
               property bool isEditing: form.state === 'Edit'
               property bool isEnabled: !!AttributeEditable && form.state !== 'ReadOnly' && !(Type === 'relation' && form.model.featureModel.modelMode == FeatureModel.MultiFeatureModel)
+              property bool notEditableInEditMode: !AttributeEditable && form.state === 'Edit'
```

Comment:
This is not a good property name here :)

I'd change the property bool isEditing condition to be form.state !== 'ReadOnly', and rename the property bool isEnabled to property bool isEditable with the following condition: 

!!AttributeEditable && !(Type === 'relation' && form.model.featureModel.modelMode == FeatureModel.MultiFeatureModel)

Then you can drop notEditableInEditMode and simply use and much clearer isEditing && isEditable condition.